src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,138 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_SIZE = 10;
+    private Object[] arrayList = new Object[INITIAL_SIZE];
+    private int arraySize;
+
+    public void grow(int size) {
```

Comment:
```suggestion
    public void grow() {
```
тобі не потрібно передавати змінну в метод так як ти маєш доступ до arrayList.length одразу тут